test/api_addProductImage_test.dart

```diff
@@ -10,10 +12,14 @@ void main() {
   const User user = TestConstants.TEST_USER;
 
   /// Common constants for several image operations
-  const String barcode = '4250752200784';
   const OpenFoodFactsLanguage language = OpenFoodFactsLanguage.GERMAN;
   const ImageField imageField = ImageField.FRONT;
 
+  /// Use a random barcode so that we can create a new product
+  /// and really upload images
+  var r = Random();
```

Comment:
Could you specify the variable type, instead of the `var` keyword please?

Reply:
is String barcode = (50000000000000 + Random().nextInt(100000000)).toString(); ok ?

Reply:
My point here was just to change the var keyword with the type of Variable

`var r = Random();`
⬇️
`Random r = Random();`